sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -69,7 +69,7 @@ def _get_resourceOp_template(op: BaseOp,
     template = {
         'apiVersion': tekton_api_version,
         'kind': 'Task',
-        'metadata': {'name': name},
+        'metadata': {'name': name, 'labels': {'app': 'resourceOp'}},
```

Comment:
@ckadner Does the new pipelineRun design allow adding labels to individual task?

Reply:
I added a labels for all the resourceOp tasks to distinguish resource and normal container. @ckadner, please let me know if not meet your requirements, or provide me a suggestion. Thanks! : )

Reply:
no, with embedded `taskSpec` `metedata` is not allowed as far as I know

@afrittoli -- do you have any advice about enabling `metadata` for embedded `taskSpec`s?

Reply:
Drop the lables for resourceOp. Resolve this issue